addons/event_sale/models/sale_order.py

```diff
@@ -23,6 +23,10 @@ def action_confirm(self):
                     .with_context(default_sale_order_id=so.id) \
                     .for_xml_id('event_sale', 'action_sale_order_event_registration')
         return res
+    
+    def action_cancel(self):
+        self.mapped('order_line')._cancel_associated_registrations()
+        super(SaleOrder, self).action_cancel()
```

Comment:
you should probably return super() otherwise you may break returned values (notably in master as we may return a wizard - to check at forward port)

Reply:
Done, thanks !

Reply:
Well my point was not about the shortened super() (which I don't really like) but the missing "return" :D

Reply:
Ok, should I do the same for the action_unlink just below?

Reply:
Always better to add a return indeed, as it returns True and not None. I think unlink is less annoying but let us propagate return value quand même.

Reply:
ok, done ;-)